src/main/resources/db/changesets/20221014_duosfirecloudgroup.yaml

```diff
@@ -0,0 +1,38 @@
+databaseChangeLog:
+  - changeSet:
+      id: duosfirecloudgroup
+      author: okotsopo
+      changes:
+        - createTable:
+            tableName: duos_firecloud_group
+            columns:
+              - column:
```

Comment:
general practice: do we want to include an id field?

Reply:
I didn't add one because the DUOS dataset ID is the primary key, and an additional identifier seemed superfluous.  Would you prefer I add one anyway?

Reply:
I would.  We do a similar thing with datasets, snapshots, etc. and it can be helpful if we want to ever support a case where there's a rename (not likely but thinking a head and figured it'd be good for consistency)

Reply:
Can do, thanks for the historical context!